docker/master/Dockerfile

```diff
@@ -26,7 +26,8 @@ RUN apt-get update && \
       python2.7 \
       unzip \
       xz-utils \
-      zlib1g-dev && \
+      zlib1g-dev \
+      patchelf && \
```

Comment:
nit: please keep the list in alphabetical order

Reply:
modified